en/admin/control.rst

```diff
@@ -2142,3 +2162,358 @@ The following shows how to use the CUBRID Manager (hereafter, CM) Administrator
+
+Starting CUBRID Java SP Server
+------------------------------
+
+The following example shows how to start CUBRID Java SP server for *demodb*.
+
+To start the Java SP server, the java_stored_procedure parameter in the CUBRID configuration file (cubrid.conf) must set to yes.
+
+::
+
+    % cubrid javasp start demodb
+
+    @ cubrid javasp start: demodb
+    ++ cubrid javasp start: success
+
+The following message is returned if CUBRID Java SP server is already running. 
+
+::
+
+    % cubrid javasp start demodb
+
+    @ cubrid javasp start: demodb
+    ++ cubrid javasp 'demodb' is running.
+
+For details on other types of errors that may occur when starting the server, see :ref:`cubrid-javasp-server-errors`.
+
+Stopping CUBRID Java SP Server
+------------------------------
+
+The following example shows how to stop CUBRID Java SP server for *demodb*. 
+
+::
+
+    % cubrid javasp stop demodb
+
+    @ cubrid javasp stop: demodb
+    ++ cubrid javasp stop: success
+
+The following message is returned when CUBRID Java SP server has stopped. 
+
+::
+
+    % cubrid javasp stop demodb
+
+    @ cubrid javasp stop: demodb
+    ++ cubrid javasp 'demodb' is not running.
+    ++ cubrid javasp stop: fail
+
+Restarting CUBRID Java SP Server
+--------------------------------
+
+The following example shows how to restart CUBRID Java SP server for *demodb*. the server that has already run stops and the server restarts. 
+
+::
+
+    % cubrid javasp restart demodb
+    
+    @ cubrid javasp stop: demodb
+    ++ cubrid javasp stop: success
+    @ cubrid javasp start: demodb
+    ++ cubrid javasp start: success
+
+Checking CUBRID Java SP Server Status
+-------------------------------------
+
+The following example shows how to check the status of a CUBRID Java SP server for *demodb*. 
+The name of Java SP server, which currently running, *demodb* is displayed. 
+Additionally, The server's PID, port number, and the applied JVM option are shown together.
+
+::
+
+    % cubrid javasp status demodb
+    
+    @ cubrid javasp status: demodb
+    Java Stored Procedure Server (demodb, pid 9220, port 38408)
+    Java VM arguments :
+    -------------------------------------------------
+    -Djava.util.logging.config.file=/path/to/CUBRID/java/logging.properties
+    -Xrs
+    -------------------------------------------------
+
+.. _cubrid-javasp-server-config:
+
+Configuring for CUBRID Java SP Server
+-------------------------------------
+
+.. _cubrid-javasp-environment-configuration:
+
+Environment Configuration for Java Stored Function/Procedure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use Java-stored functions/procedures in CUBRID, you must have JRE (Java Runtime Environment) 1.6 or better installed in the environment where the CUBRID server is installed. You can download JRE from the Developer Resources for Java Technology (`https://www.oracle.com/java/technologies <https://www.oracle.com/java/technologies>`_).
```

Comment:
propose: better --> later
for version, commonly use "later"

Reply:
Thank you, I'll fix it